src/components/ViolationForm.js

```diff
@@ -0,0 +1,357 @@
+          <p className="errorMsg">Полето е задължително.</p>
+        )}
+      </div>
+      <div>
+        {selectedTown ? (
+          getTownById(selectedTown)[0].cityRegions.length != 0 ? (
+            <div>
+              <label className="inputLabel">Квартал</label>
+              <select
+                className="form-control"
+                name="city_region"
+                {...register('city_region', { required: true })}
+                onChange={(e) => setSelectedCityRegion(e.target.value)}
+              >
+                <option value="" disabled selected="selected">
+                  -- Квартали --
+                </option>
+                {getCityRegions()}
+              </select>
+            </div>
+          ) : (
+            <div> not</div>
+          )
+        ) : (
+          <div></div>
+        )}
+      </div>
+      <div className="form-control">
+        <label className="inputLabel">Име</label>
+        <input
+          type="text"
+          name="name"
+          {...register('name', { required: true })}
+        />
+        {errors.name && errors.name.type === 'required' && (
+          <p className="errorMsg">Полето е задължително.</p>
+        )}
+      </div>
+      <div className="form-control">
+        <label className="inputLabel">Имейл</label>
+        <input
+          type="text"
+          name="email"
+          {...register('email', {
+            required: true,
+            pattern: {
+              value:
+                /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+              message: 'Въведете валиден имейл',
+            },
+          })}
+        />
+        {errors.email && errors.email.type === 'required' && (
+          <p className="errorMsg">Полето е задължително.</p>
+        )}
+        {errors.email && errors.email.message && (
+          <p className="errorMsg">{errors.email.message}</p>
+        )}
+      </div>
+      <div className="form-control">
+        <label className="inputLabel">Телефон</label>
+        <input
+          type="text"
+          name="phoneNumber"
+          {...register('phoneNumber', { required: true })}
+        />
+        {errors.phoneNumber && errors.phoneNumber.type === 'required' && (
+          <p className="errorMsg">Полето е задължително.</p>
+        )}
+      </div>
+      <div className="form-control">
+        <label className="inputLabel">Секция</label>
+        <input
+          type="text"
+          name="section"
+          {...register('section', {
+            required: false,
+            pattern: {
+              value: /^(0|[1-9]\d*)(\.\d+)?$/,
+              message: 'Забранено въвеждането на текст',
+            },
+          })}
+        />
+        {errors.section && errors.section.message && (
+          <p className="errorMsg">errors.section.message</p>
```

Comment:
Показва ми това съобщение когато се опитвам да подам сигнал със секция. 
Предполагам, че трябва да е:
```suggestion
          <p className="errorMsg">{{ errors.section.message }}</p>
```